spec/index.md

```diff
@@ -305,13 +278,19 @@ Each dimension is represented by a hash in the `dimensions` array. The hash has
     "primaryKey": ["Project", "ClassCode"],
 
     // OPTIONAL: Describes what kind of a dimension it is. `dimensionType` is a string that `MUST` be one of the following:
-    // datetime, entity, classification, activity, fact, location, other`
-    "dimensionType": "classification",
+    // * "datetime": the date of a transaction 
+    // * "entity": names the organisation doing the spending or receiving
+    // * "classification": one or more fields that create a categorical hierarchy of the type of spending (eg, Health > Hospital services > Nursing) 
+    // * "activity": one or more fields that create an administrative hierarchy (eg, Department > Programme > Project)
```

Comment:
"Activity" is meant to denote a dimension/hierarchy related to programmes and projects which may have start and end dates and other characteristics that separate them from other types of budget classifications (https://github.com/openspending/fiscal-data-package/issues/4#issuecomment-149537398).

An "administrative" dimension/hierarchy may have some overlap with a programme hierarchy, but is meant to be separate.  Compare pages 6 and 10 here: https://www.imf.org/external/pubs/ft/tnm/2009/tnm0906.pdf